cosmwasm/packages/wasmi-runtime/src/crypto/kdf.rs

```diff
@@ -67,8 +67,8 @@ pub mod tests {
     use super::{
         Keychain, CONSENSUS_SEED_SEALING_PATH, KEY_MANAGER, REGISTRATION_KEY_SEALING_PATH,
     };
+    use crate::crypto::errors::CryptoError;
```

Comment:
Might be cleaner to export it directly from crypto, so we can have a single import from crate::crypto

Reply:
good idea

Reply:
done